app/views/alchemy/essences/_essence_headline_view.html.erb

```diff
@@ -0,0 +1 @@
+<%= content_tag "h#{content.essence.level}", content.essence.ingredient, class: "h#{content.essence.size} #{options[:classes]}" %>
```

Comment:
1. We dont always have a `size`, this way we would end up with a class `H`
2. It should be `options[:class]` to be conform wih Rails helpers and all other Alchemy essences
3. We want to avoid to set `class` to a "three empty spaces" string if we not have a `size` and/or `class` options.